devicesetup/src/main/res/values/themes.xml

```diff
@@ -26,7 +27,8 @@
         <item name="android:textAppearanceSmallPopupMenu">@style/PopupMenuText</item>
     </style>
 
-    <style name="PopupMenu" parent="Widget.AppCompat.Light.PopupMenu.Overflow">
+    <style name="PopupMenu" parent="Widget.AppCompat.Light.PopupMenu.Overflow"
```

Comment:
Weirdly after I deleted warning suppression there was no lint error as before. Nonetheless I copied "Widget.AppCompat.Light.PopupMenu.Overflow" style into our themes.xml just in case.